client/src/components/Leaflet.js

```diff
@@ -130,13 +131,14 @@ const Leaflet = ({
         }
         if (m.onMove) {
           marker.on("move", event => m.onMove(event, map))
+          map.on("click", event => marker.setLatLng(event.latlng).update())
         }
         markerLayer.addLayer(marker)
       })
 
       if (newMarkers.length > 0) {
         if (markerLayer.getBounds() && markerLayer.getBounds().isValid()) {
-          map.fitBounds(markerLayer.getBounds(), { maxZoom: 15 })
+          map.fitBounds(markerLayer.getBounds(), { maxZoom: maxZoom })
```

Comment:
Prefer the more compact notation of:
```
          map.fitBounds(markerLayer.getBounds(), { maxZoom })
```
when the variable name is identical to the prop.